src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
you use the injector only for services, and this annotation is only for hibernate 
```suggestion
@Column(unique = true)
    private String email;
```